test/integration/targets/vyos_interfaces/vars/main.yaml

```diff
@@ -0,0 +1,233 @@
+---
+merged:
+  before:
+    - name: "eth0"
+      enable: true
+      speed: "auto"
+      duplex: "auto"
+
+    - name: "eth1"
+      enable: true
+
+    - name: "eth2"
+      enable: true
+
+    - name: "lo"
+      enable: true
+
+  commands:
+    - "set interfaces ethernet eth1 description 'Configured by Ansible - Interface 1'"
+    - "set interfaces ethernet eth1 mtu '1500'"
+    - "set interfaces ethernet eth1 duplex 'auto'"
+    - "set interfaces ethernet eth1 speed 'auto'"
+    - "set interfaces ethernet eth1 vif 100"
+    - "set interfaces ethernet eth1 vif 100 description 'Eth1 - VIF 100'"
+    - "set interfaces ethernet eth1 vif 100 mtu '400'"
+    - "set interfaces ethernet eth1 vif 101"
+    - "set interfaces ethernet eth1 vif 101 description 'Eth1 - VIF 101'"
+    - "set interfaces ethernet eth2 description 'Configured by Ansible - Interface 2 (ADMIN DOWN)'"
+    - "set interfaces ethernet eth2 mtu '600'"
+    - "set interfaces ethernet eth2 disable"
+
+  after:
+    - name: "lo"
+      enable: true
+
+    - name: "eth0"
```

Comment:
here too

Reply:
This is not a test. This is the fixtures file.